lib/std/target.zig

```diff
@@ -649,6 +650,13 @@ pub const Target = union(enum) {
             }
         }
 
+        if (self.isWasm()) {
+            switch (self.getArchPtrBitWidth()) {
+                32 => return Executor{ .wasmtime = "wasmtime" },
```

Comment:
I'm curious... have you tried this and it didn't worked? @fengb

```
                32 => return Executor{ .wasi = "wasmer run" },
```

Reply:
The args are generated separately `--test-cmd wasmer --test-cmd run` so I'd have to update the plumbing to support multiple args.

Reply:
That make sense! Thanks for the info.

To support this use case, we are working now on allowing to pass a file as first argument without being explicit with `run` (so you can do `wasmer myfile.wasm`).
I'll post an update here once it's ready to be used (which should be very soon!)

Reply:
We just added support for running wasm files directly without the `run` argument.
https://github.com/wasmerio/wasmer/pull/990

We will publish a release soon :)